backend/src/core/models/steam.app.raw.js

```diff
@@ -0,0 +1,31 @@
+export class SteamAppRaw {
+  id;
+  name;
+  type;
+  developers;
+  genres;
+  description;
+  releaseDate;
+
+  constructor(data) {
+    this.id = data.steam_appid;
+    this.name = data.name;
+    this.type = data.type;
+    this.developers = data.developers;
+    this.genres = this.#extractGenres(data.genres);
+    this.description = data.short_description;
+    this.releaseDate = this.#extractReleaseDate(data.release_date);
+  }
+
+  #extractGenres(genres) {
+    if (!genres) return [];
+
+    return genres.map((genre) => genre.description);
+  }
+
+  #extractReleaseDate(releaseDate) {
+    if (!releaseDate) return "";
```

Comment:
Don't forget to change this to undefined or null. Dates should not have empty string as a null value.